package.json

```diff
@@ -30,7 +30,9 @@
   "dependencies": {
     "@momentum-ui/react": "^23.21.4",
     "@webex/components": "^1.45.6",
-    "@webex/sdk-component-adapter": "^1.24.1"
+    "@webex/sdk-component-adapter": "^1.24.1",
+    "@webex/test-users": "^1.119.4",
```

Comment:
`@webex/test-users` and `dotenv` should not be dependencies. Please make them dev dependencies